include/rnp/rnp.h

```diff
@@ -653,14 +687,43 @@ RNP_API rnp_result_t rnp_import_keys(rnp_ffi_t   ffi,
                                      uint32_t    flags,
                                      char **     results);
 
-/** import standalone signatures to the keyring and receive JSON list of the updated keys.
+/** import standalone signatures to the keyring and receive JSON list of the updated
+ * signatures.
  *
  *  @param ffi
  *  @param input source to read from. Cannot be NULL.
  *  @param flags additional import flags, currently must be 0.
  *  @param results if not NULL then after the successful execution will contain JSON with
- *                 information about the updated keys. You must free it using the
+ *                 information about the updated signatures. You must free it using the
  *                 rnp_buffer_destroy() function.
+ *                 JSON output is an object containing array of objects named "sigs".
+ *                 Each array item is an object representing imported signature.
+ *                 It contains the following members:\n
+ * JSON member         | Description
+ * --------------------|------------
+ * "public"            |string, status of a public key signature
```

Comment:
It should be better rephrased to something mentioning 'status of import to the public keyring/to the secret keyring'

Reply:
"import status of a public key signature"?

Reply:
> "import status of a public key signature"?

It's a bit misleading since there are no distinction between public key and secret key signature, it's the same. The only difference is whether signature succesfully written to the corresponding keyring.

Reply:
"signature import status in a public/secret keyring"?

Reply:
@antonsviridenko Sounds good.